modules/learn/pages/clusters-and-availability/xdcr-viewing-conflict-logs.adoc

```diff
@@ -0,0 +1,356 @@
+    copy[meta.id] = doc;
+    }
+}
+----
+
+Resultant document example:
+
+NOTE: See xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#conflict-record-document-format[Conflict Record Document Format] to understand the document format.
+
+----
+{
+  "timestamp": "2025-05-20T19:25:55.308Z",
+  "id": "crd_1747769155308581666_c0468a42632daabce649b1fb7e174a530946c04b1dde75218290c4885ad11669",
+  "docId": "ASHUNK100",
+  "replId": "1bc375a26b7f822ebe91288c58a6c614/bucket1/bucket1-PN8-oFADADCOe6S6A8YuIw==",
+  "srcDoc": {
+    "id": "src_1747769155308581666_c0468a42632daabce649b1fb7e174a530946c04b1dde75218290c4885ad11669",
+    "nodeId": "10.0.13.10",
+    "bucketUUID": "6cd3a35ba2d337d84d4fcd30c3a46fb9",
+    "clusterUUID": "d976db26a5ce512a243db65a3026073f",
+    "isDeleted": false,
+    "collection": "_default",
+    "scope": "_default",
+    "expiry": 0,
+    "flags": 33554438,
+    "cas": 1747769124557291500,
+    "revSeqno": 4,
+    "xattrs": {
+      "_vv": "{\"cvCas\":\"0x00002a1777524118\",\"src\":\"ntR6bcR9VHcjcPpQ5osEiw\",\"ver\":\"0x00002a1777524118\",\"pv\":[\"000018fe76524118@8DdrZBqHaACpmDeYJIv+kg\"]}",
+      "_sync": "",
+      "_mou": ""
+    }
+  },
+  "tgtDoc": {
+    "id": "tgt_1747769155308581666_c0468a42632daabce649b1fb7e174a530946c04b1dde75218290c4885ad11669",
+    "nodeId": "10.0.22.102",
+    "bucketUUID": "fada12469f1027ecb078d2c19a2802fb",
+    "clusterUUID": "1bc375a26b7f822ebe91288c58a6c614",
+    "isDeleted": false,
+    "collection": "_default",
+    "scope": "_default",
+    "expiry": 0,
+    "flags": 33554438,
+    "cas": 1747769104359030800,
+    "revSeqno": 4,
+    "xattrs": {
+      "_vv": "{\"cvCas\":\"0x000018fe76524118\",\"src\":\"8DdrZBqHaACpmDeYJIv+kg\",\"ver\":\"0x000018fe76524118\"}",
+      "_sync": "",
+      "_mou": ""
+    }
+  },
+"src_doc": {  
+    "author": "Unknown",  
+    "create_date_utc": "2025-05-20T07:56:26",  
+    "res_from": "WEST",  
+    "res_name": "Harry West",  
+    "res_note": "visitor",  
+    "reserved": true,  
+    "title": "Ashmole History",  
+    "type": "book",  
+    "unix_time": "1747768801",  
+    "update_date_utc": "2025-05-20T19:20:01"  
+  },
+  "tgt_doc": {
+    "author": "Unknown",
+    "create_date_utc": "2025-05-20T07:56:26",
+    "res_from": "WEST",
+    "res_name": "Harry West",
+    "res_note": "visitor -- HJV NOTE WEST",
+    "reserved": true,
+    "title": "Ashmole 782",
+    "type": "book",
+    "unix_time": "1747768801",
+    "update_date_utc": "2025-05-20T19:20:01"
+  }
+}
+----
+
+[#using-kv-range-scan-clog]
+== Using KV Range Scan
+
+To use only KV, use Couchbase SDK with KV APIs including xref:8.0@server:howtos/kv-operations.adoc#kv-range-scan[KV Range Scan].
+
+For example, to get a list of all `crd` prefix documents (CRD documents) using the KV Range Scan xref:8.0@server:howtos/kv-operations.adoc#kv-range-scan-prefix[PrefixScan], use KV `collection.get` to get the conflicting `src` and `tgt` docs.
```

Comment:
Replace `crd` with `crd_`
Replace `src` with `src_`
Replace `tgt` with `tgt_`

Reason is that prefixes/strings like "crd", "src", and "tgt" are used in other places and as part of other field names.  So, putting the '_' part distinguishes them as CRD document id's.